transports/sqs/delayed-delivery.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Delayed Delivery
+summary: Describes the native delayed delivery implementation in the SQS transport
+component: Sqs
+reviewed: 2018-03-02
+versions: '[4,]'
+---
+
+In Versions 4.0 and above, the SQS transport allows [delayed delivery](/nservicebus/messaging/delayed-delivery.md) of messages longer than 15 minutes. The transport creates a FIFO queue per endpoint that allows delaying messages for longer periods of time.
+
+## Enable unrestricted delayed delivery
+
+The unrestricted delayed delivery has to be enabled on the transport configuration:
+
+snippet: DelayedDelivery
+
+Unrestricted delayed delivery needs to be enabled both on the sender and receiver to be able to process delayed deliveries longer than 15 minutes (900 seconds). Delayed delivery below 15 minutes is always supported. The following table illustrates that.
+
+| Scenario                    | Sender   | Receiver | Supported     |
+|-----------------------------|----------|----------|---------------|
+| Delayed Delivery <= 900 sec | disabled | disabled | Yes           |
+|                             | disabled | enabled  | Yes           |
+|                             | enabled  | disabled | Yes           |
+|                             | enabled  | enabled  | Yes           |
+| Delayed Delivery > 900 sec  | disabled | disabled | No            |
+|                             | disabled | enabled  | No            |
+|                             | enabled  | disabled | No            |
+|                             | enabled  | enabled  | Yes           |
+
+Enabling the unrestricted delayed delivery will require a FIFO queue to be created for each endpoint that receives delayed deliveries. The FIFO queue follows a fixed naming convention by appending `-delay.fifo` to the queue name of the endpoint. The creation of the FIFO queue requires the [installers](/nservicebus/operations/installers.md) to be enabled or the queue being created upfront via [scripting](/transports/sqs/operations-scripting.md).
```

Comment:
Should we state here that the FIFO queue delay has to be set to 15 minutes? Because we expect it to be 15 minutes.